blueprints/app/index.js

```diff
@@ -32,6 +32,7 @@ module.exports = {
         [!options.welcome && '"--no-welcome"', options.yarn && '"--yarn"'].filter(Boolean).join(',\n            ') +
         outdent;
     }
+    let lang = options.lang;
```

Comment:
I'd vaguely prefer for this to follow suit with the style below. We either need to update `welcome: options.welcome` and `yarn: options.yarn` to destructure, or change this one to use `lang: options.lang`. Either way seems totally fine to me.

Reply:
Thanks for spotting this! I destructured it before the utilities and template logic were consolidated and forgot to revert it once I finished. I can absolutely update `lang` to be consistent with `yarn` and `welcome`.